Formula/jenv.rb

```diff
@@ -15,12 +15,24 @@ def install
   end
 
   def caveats
-    <<~EOS
-      To activate jenv, add the following to your #{shell_profile}:
+    if preferred == :fish
+      s = <<~EOS
+        To activate jenv, run the following commands:
 
-        export PATH="$HOME/.jenv/bin:$PATH"
-        eval "$(jenv init -)"
-    EOS
+      EOS
+      s << "  fish_add_path #{bin}\n" if bin.directory?
```

Comment:
I originally had

```
      s << "  #{Utils::Shell.prepend_path_in_profile(bin)}\n" if bin.directory?
```

which outputs

```
echo 'set -g fish_user_paths "/opt/homebrew/opt/jenv/bin" $fish_user_paths' >> ~/.config/fish/config.fish
```

But current fish docs here \[2\] say

> A faster way is to use the `fish_add_path function`...
> The advantage is that you don't have to go mucking around in files:
> just run this once at the command line, and it will affect the current session and all future instances too.
>
> Or you can modify `$fish_user_paths` yourself, but you should be careful not to append to it
> unconditionally in config.fish, or it will grow longer and longer.

`set -g fish_user_paths "/opt/homebrew/opt/jenv/bin" $fish_user_paths` seems to be doing just that: appending unconditionally. But I haven't observed my `$PATH` growing longer and longer either.

In any case, that's why I put the line above. Related issue https://github.com/Homebrew/brew/issues/8965.
I'll submit a PR for that issue. If that gets merged, I'll open another PR here to update this line to use the utility function.

2: https://fishshell.com/docs/current/tutorial.html#path